packages/manager/modules/sign-up/src/form/details/details.constants.js

```diff
@@ -245,6 +245,9 @@ export const PHONE_PREFIX = {
   ZW: '263',
 };
 
+export const PURPOSE_OF_PURCHASE_MAX_LENGTH = 255;
```

Comment:
isn't it provide by api /newAccount/rules ?

Reply:
no I dont see it.

Reply:
Can you check this with Sulliac please?

Reply:
I check on my side with Alexandre Legent (responsible to update the API for purposeOfPurchase)
Indeed the max length of the field is not provided by the API, but we agreed that it should as the API will refuse a value longer than 255 characters.
A fix will be made at some point to handle it (and also the address which has the same restriction)